libs/pageserver_api/src/models.rs

```diff
@@ -162,6 +162,14 @@ impl std::fmt::Debug for TenantState {
     }
 }
 
+#[serde_as]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct LsnLease {
+    #[serde(rename = "valid_until_millis_since_epoch")]
+    #[serde_as(as = "serde_with::TimestampMilliSeconds")]
+    pub valid_until: SystemTime,
+}
```

Comment:
Let's document this right away, document as in add type level docs. The `models.rs` is a garbage bin of types with them rarely having any docs, but we can do better for new types.

Reply:
docs added.